public/locales/en/filters.json

```diff
@@ -6,18 +6,19 @@
   "collegeUniversity": "College/University",
   "deadlineEarliestToLatest": "Deadline (Earliest to Latest)",
   "deadlineLatestToEarliest": "Deadline (Latest to Earliest)",
-  "enterMajorFilter": "Enter a major to filter by",
-  "enterStateFilter": "Enter a state to filter by",
   "filters": "Filters",
   "filtersCurrentlyApplied": "Your filters are currently applied.",
   "gradeLevel": "Grade Level",
   "highSchool": "High School",
   "limitReached": "Limit reached",
+  "major": "major",
```

Comment:
Should this be capitalized like the other two?

I wonder if we should formally establish a paradigm for the capitalization of single words

Reply:
The reason the others are capitalize is because they are used as the Filter title and major is only used for the placeholder.

Reply:
I'll capitilize though, for consistency reasons.